packages/@sparrow-workspaces/src/components/test-flow-tour-guide/TestFlowTourGuide.svelte

```diff
@@ -0,0 +1,216 @@
+<script>
+  import { CrossIcon } from "@sparrow/library/assets";
+  import { onMount, onDestroy } from "svelte";
+  import PulseCircle from "../../../../@sparrow-common/src/components/pulse-circle/PulseCircle.svelte";
```

Comment:
instead of like this import like @sparrow-common/src/components

Reply:
Resolved